src/Evolution/DgSubcell/SubcellOptions.hpp

```diff
@@ -200,13 +200,24 @@ class SubcellOptions {
     using group = FdToDgTci;
   };
 
+  /// \brief FD to FD interpolation order.
+  ///
+  struct FdInterpolationOrder {
```

Comment:
Wait, doesn't this have to be in the same commit as the input file updates? Can you make sure each commit compiles and passes tests?